src/contracts/UpgradeRegressionManager.sol

```diff
@@ -110,11 +111,8 @@ contract UpgradeRegressionManager is IUpgradeRegressionManager {
   /// @notice Time window after a rollback is proposed during which it can be queued for execution.
   uint256 public rollbackQueueWindow;
 
-  /// @notice Timestamp after which rollback queueing is no longer allowed.
-  mapping(uint256 rollbackId => uint256 deadline) public rollbackQueueExpiresAt;
-
-  /// @notice Time after which the rollback can be executed.
-  mapping(uint256 rollbackId => uint256 eta) public rollbackExecutableAt;
+  /// @notice Rollback id to rollback data.
+  mapping(uint256 rollbackId => Rollback) private _rollbacks;
```

Comment:
Even though this is private, we should not use the `_` prefix here. Our convention is that `_` implies a *non*-storage access, so including it here causes confusion.